app/move/controllers/create/move-details.test.js

```diff
@@ -197,86 +197,110 @@ describe('Move controllers', function() {
+                    {
+                      text: 'Albany',
+                      value: '1',
+                    },
+                    {
+                      text: 'Altcourse',
+                      value: '2',
+                    },
+                  ],
+                })
+              })
+              it('appends the correct conditional', function() {
+                expect(req.form.options.fields.additional_information).not.to
+                  .exist
+                expect(referenceDataService.getLocationsByType).to.have.been
+                  .calledTwice
+              })
+              it('calls the location service with the correct type of location', function() {
+                expect(
+                  referenceDataService.getLocationsByType
+                    .getCall(0)
+                    .calledWithExactly('court')
+                ).to.be.true
+                expect(
+                  referenceDataService.getLocationsByType
+                    .getCall(1)
+                    .calledWithExactly('prison')
+                ).to.be.true
+              })
             }
-            await controller.configure(req, res, nextSpy)
-            expect(req.form.options.fields.move_type).to.deep.equal({
-              items: [
-                {
-                  value: 'court_appearance',
-                },
-                {
-                  value: 'prison_recall',
-                  text: '__translated__',
-                  conditional: 'to_location_prison_recall',
-                },
-              ],
-            })
-            expect(
-              req.form.options.fields.to_location_prison_recall
-            ).to.deep.equal({
-              items: [
-                {
-                  text: '--- Choose prison ---',
-                },
-                {
-                  text: 'Albany',
-                  value: '1',
-                },
-                {
-                  text: 'Altcourse',
-                  value: '2',
-                },
-              ],
-            })
-            expect(req.form.options.fields.additional_information).not.to.exist
-            expect(referenceDataService.getLocationsByType).to.have.been
-              .calledTwice
-            expect(
-              referenceDataService.getLocationsByType
-                .getCall(0)
-                .calledWithExactly('court')
-            ).to.be.true
-            expect(
-              referenceDataService.getLocationsByType
-                .getCall(1)
-                .calledWithExactly('prison')
-            ).to.be.true
-          })
-          it('when the current location type is police appends additional_information as conditional', async function() {
-            req.session = {
-              currentLocation: {
-                location_type: 'police',
-              },
+          )
+          context(
+            'when the current location type is police appends additional_information as conditional',
```

Comment:
Same as the other context. Purpose is just to set the scene, not describe any outcomes. And can also remove `async` from the function defintion.